main.go

```diff
@@ -106,6 +106,68 @@ func signTxCmd() *ffcli.Command {
 	}
 }
 
+func batchSignTxCmd() *ffcli.Command {
+	fs := flag.NewFlagSet("sign-batch", flag.ContinueOnError)
```

Comment:
Since the first sign command is called `sign-tx`, maybe rename this one `sign-batch-tx` or `sign-tx-batch` ?